Makefile

```diff
@@ -209,13 +213,10 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 ##@ Templates
 
 .PHONY: cluster-e2e-templates
-cluster-e2e-templates: cluster-e2e-templates-v1beta1 cluster-e2e-templates-v1alpha4 cluster-e2e-templates-v124 cluster-e2e-templates-v130 ## Generate cluster templates for all versions
-
-cluster-e2e-templates-v124: ## Generate cluster templates for CAPX v1.2.4
-	kustomize build $(NUTANIX_E2E_TEMPLATES)/v1.2.4/cluster-template --load-restrictor LoadRestrictionsNone > $(NUTANIX_E2E_TEMPLATES)/v1.2.4/cluster-template.yaml
+cluster-e2e-templates: cluster-e2e-templates-v1beta1 cluster-e2e-templates-v1alpha4 cluster-e2e-templates-v135 ## Generate cluster templates for all versions
 
-cluster-e2e-templates-v130: ## Generate cluster templates for CAPX v1.3.0
-	kustomize build $(NUTANIX_E2E_TEMPLATES)/v1.3.0/cluster-template --load-restrictor LoadRestrictionsNone > $(NUTANIX_E2E_TEMPLATES)/v1.3.0/cluster-template.yaml
+cluster-e2e-templates-v135: ## Generate cluster templates for CAPX v1.3.0
+	kustomize build $(NUTANIX_E2E_TEMPLATES)/v1.3.5/cluster-template --load-restrictor LoadRestrictionsNone > $(NUTANIX_E2E_TEMPLATES)/v1.3.5/cluster-template.yaml
```

Comment:
CAPX v1.3.0 did not support K8s 1.29+ as such we had to switch to the latest patch version in CAPX v1.3.x train.